README.md

```diff
@@ -17,7 +17,11 @@
 ## Creating development environment
 ### Prerequisites
 1. Windows or Linux based system - either [WSL on windows](https://docs.microsoft.com/en-us/windows/wsl/install-win10) or full blown linux.
-2. [Python](https://www.python.org/downloads/release/python-385/) 
+2. [Python](https://www.python.org/downloads/release/python-385/)
+3. Install pre-commit hooks:
+```shell
+pre-commit install
```

Comment:
This comment doesn't work on my shell. Maybe it requires some another prerequisite?

Reply:
I changed it and I think it will work for you (and everyone else too) now :)

Reply:
![image](https://user-images.githubusercontent.com/1492939/107499781-0b171900-6b9e-11eb-9425-71708089f8b4.png)
^^"

Reply:
but I change it, ok now it is correct
you need to install the requirements before

Reply:
Maybe you've changed the requirements.txt and forgot to push it?

Currently I don't have anything in the requirements.txt that installs the `pre-commit` command, unfortunately :(

Reply:
oh no, your right, my mistake
I fixed it now